app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,12 @@
+<% provide(:title, t(".title")) %>
+<h1><%= t ".heading" %></h1>
+<p>
+  <%= t ".paragraph1" %><a href="https://www.railstutorial.org/"><em>
+    <%= t ".emphasized" %></em></a> <%= t ".paragraph2" %>
```

Comment:
indent hì

thay vì dùng tag `a` thì dùng `link_to` của rails nhé
SỬA TOÀN BỘ FILE HTML NHÉ